src/configure/clients/devOps/azureDevOpsClient.ts

```diff
@@ -136,7 +134,7 @@ export class AzureDevOpsClient {
     public async getRepository(organizationName: string, projectName: string, repositoryName: string): Promise<any> {
         let url = `${AzureDevOpsBaseUrl}/${organizationName}/${projectName}/_apis/git/repositories/${repositoryName}`;
 
-        return this.sendRequest(<UrlBasedRequestPrepareOptions>{
+        return this.sendRequestWithMsaOrgCheck(<UrlBasedRequestPrepareOptions>{
```

Comment:
can we rather cache this in this file (for the org chosen), we are making a call each time otherwise